packages/frontend/src/components/FlowStepTestController/utils.tsx

```diff
@@ -136,26 +139,68 @@ export const matchParamsToDataIn = (
       }
 
       const match = String(paramValue).match(STEP_ID_REGEX)
-      const searchKey = match?.[0]
+      let isFormSgTable = false
```

Comment:
nit: 
```suggestion
      const isFormSgTable = String(paramValue).includes('fields') &&
        String(paramValue).includes('answer')
```

but is this the best way to check if it's formsg table? can look at inputSource from lasttest?

Reply:
thanks for cathing, looking for `inputSource` and `formsg-table` is a much better way to check this